baekjoon/피보나치-수-5.js

```diff
@@ -0,0 +1,10 @@
+const n = parseInt(require("fs").readFileSync("/dev/stdin").toString());
+
+function solution(n) {
+    for (let i = 1; i <= n; i++) {
+        n = solution(n - 1) + solution(n - 2);
+    }
+    return n;
+}
```

Comment:
```suggestion
function fibonacci(n) {
  if (n === 0) return 0;
  if (n === 1) return 1;
  return fibonacci(n - 1) + fibonacci(n - 2);
}
```
이미 재귀함수를 사용해서 루프를 돌리고 있는데 굳이 또 반복문을 사용하는건 제 생각엔 별로 좋아보이지 않네요
그리고 개인적인 생각으로 함수명을 solution보다는 fibonacci 로 하면 어떨까 싶어요